SPECS/openssl/0025-for-tests.patch

```diff
@@ -8,7 +8,7 @@ diff -up openssl-3.0.0/apps/openssl.cnf.xxx openssl-3.0.0/apps/openssl.cnf
 -default = default_sect
 +##default = default_sect
  ##legacy = legacy_sect
- ##
+ #
```

Comment:
Is there a way we can annotate the original patch was modified by us?

Reply:
good idea; I added a header comment to the patch file